src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -145,7 +145,7 @@
   background: var(--pf-c-options-menu__toggle--Background);
   border: none;
 
-  > * {
+  .pf-c-options-menu__toggle-button-icon {
```

Comment:
doesn't really matter but this selector probably doesn't need to be nested in `.pf-c-options-menu__toggle`